pkg/operator/ceph/object/spec.go

```diff
@@ -1130,3 +1136,59 @@ func GetHostnameFromEndpoint(endpoint string) (string, error) {
 
 	return hostname, nil
 }
+
+// Detects Runtime OS family
+func detectRuntimeOSFamily() string {
+	file, err := os.Open("/etc/os-release")
```

Comment:
Is your intent to pick up the release version of the container or the host?

Reply:
After you pointed out I realized it has been done for case host and guest are in the same family. I will update my pr to include all the cases.

Reply:
Updated it now.